aesara/scan/op.py

```diff
@@ -3440,7 +3440,7 @@ def profile_printer(
             )
 
 
-@op_debug_information.register(Scan)
+@op_debug_information.register(Scan)  # type: ignore
```

Comment:
This happens sometimes, because the pre-commit only runs on modified files. But you need it for the CI pre-commit to pass, as it also checks the scan module properly in that case

Reply:
Although... it seems to be passing now?

Reply:
That shouldn't be failing anymore.

Reply:
pre-commit run --all is ok
once I modify files it fails with scan